cli/src/command/run_dev.rs

```diff
@@ -82,7 +82,7 @@ impl ExonumCommand for RunDev {
         let node_config_file_name = "node.toml";
 
         let finalize = Finalize {
-            secret_config_path: self.artifact_path(SEC_CONFIG_FILE_NAME),
+            private_config_path: self.artifact_path(SEC_CONFIG_FILE_NAME),
```

Comment:
Nit: The `SEC_CONFIG_FILE_NAME` name still sort of implies "secret" instead of "private."